html-templates/verified_credentials.html

```diff
@@ -289,9 +290,19 @@ <h5 v-if="state.showScanned" class="fw-bolder mb-3">
 
   <script type="text/javascript">
     const { createApp, ref, reactive, computed, watch } = Vue;
-    /**
+   /**
      * @typedef  {"intro" |"verified" | "failed" | "pending"| "expired" |"abandoned"} AppStates
      */
+    // user agent detection
+   const uap = new UAParser();
+   const getBrowserUA = () => {
+     PREFIX_MOBILE  = 'Mobile ';
+     // @type string
+     const b = uap.getResult().browser.name
+     const browserName = b.startsWith(PREFIX_MOBILE) ? b.split(" ")[1] : b.split(" ")[0]
+     return `{{wallet_deep_link}}&browser=${browserName.toUpperCase()}`;
+   }
+   console.log(`Detected Browser: ${uap.getResult().browser.name}`)
```

Comment:
Should guard this code as well since we're calling a library (so anything that tries uap.getResult().browser.name) and doing string and array access so an exception could blow the render up.
Can try-catch around and log out an error to console, and handle so that then if the browser field is just not set the deep link will still work (as the wallet will have to handle missing/blank/unknown values in that anyways)